cdap-docs/reference-manual/source/faq.rst

```diff
@@ -96,8 +96,8 @@ Yes. You can install CDAP on your Hadoop cluster. See :ref:`install`.
 
 .. rubric:: What Hadoop distributions can CDAP run on?
 
-CDAP has been tested on and supports CDH 4.2.x through 5.3.x, HDP 2.0 through 2.1, and
-Apache Bigtop 0.8.0. 
+CDAP |version| has been tested on and supports CDH 5.0.0 through 5.4.4; HDP 2.0, 2.1, and 2.2; 
```

Comment:
I have standardized on use of the "Oxford" comma, as it is consistent and reduces confusion. Otherwise, you can't tell if that is a list of three items (2.0, 2.1, and 2.2) or two items (2.0, and an item composed of 2.1 and 2.2). There's been entire books written on the subject...